cli/determined_cli/shell.py

```diff
@@ -71,17 +71,32 @@ def _open_shell(master: str, shell: Command, additional_opts: str) -> None:
         if host == LOOPBACK_ADDRESS:
             host = "localhost"
 
+        # Use determined_cli.tunnel as a portable script for using the HTTP CONNECT mechanism,
+        # similar to `nc -X CONNECT -x ...` on unix systems.
```

Comment:
highly non-blocking: netcat has been ported to Windows, so I'm guessing this statement could technically be a bit broader. Also "unix" → "Unix".

Reply:
fixed.